docs/tutorial/saxpy.rst

```diff
@@ -0,0 +1,780 @@
+    :sync: windows-nvidia
+
+    Windows compilers and command line tooling have traditionally relied on
+    extra environmental variables and PATH entries to function correctly.
+    Visual Studio refers to command lines with this setup as "Developer
+    Command Prompt" or "Developer PowerShell" for ``cmd.exe`` and PowerShell
+    respectively.
+
+    The HIP and CUDA SDKs on Windows don't include complete toolchains. You will
+    also need:
+
+    - The Microsoft Windows SDK. It provides the import libs to crucial system
+      libraries that all executables must link to and some auxiliary compiler
+      tooling.
+    - A Standard Template Library (STL). Installed as part of the Microsoft
+      Visual C++ compiler (MSVC) or with Visual Studio.
+
+    If you don't have a version of Visual Studio 2022 installed, for a
+    minimal command line experience, install the
+    `Build Tools for Visual Studio 2022 <https://aka.ms/vs/17/release/vs_BuildTools.exe>`_
+    with the Desktop Developemnt Workload. Under Individual Components select:
+
+    - A version of the Windows SDK
+    - "MSVC v143 - VS 2022 C++ x64/x86 build tools (Latest)"
+    - "C++ CMake tools for Windows" (optional)
+
+    .. note::
+
+      The "C++ CMake tools for Windows" individual component is a convenience which
+      puts both ``cmake.exe`` and ``ninja.exe`` onto the PATH inside developer
+      command prompts. You can install these manually, but then you must manage
+      them manually.
+
+    Visual Studio 2017 and later are detectable as COM object instances via WMI.
+    To setup a command line from any shell for the latest Visual Studio's
+    default Visual C++ toolset issue:
+
+    .. code-block:: powershell
+
+      $InstallationPath = Get-CimInstance MSFT_VSInstance | Sort-Object -Property Version -Descending | Select-Object -First 1 -ExpandProperty InstallLocation
+      Import-Module $InstallationPath\Common7\Tools\Microsoft.VisualStudio.DevShell.dll
+      Enter-VsDevShell -InstallPath $InstallationPath -SkipAutomaticLocation -Arch amd64 -HostArch amd64 -DevCmdArguments '-no_logo'
+
+    You should be able to call the compiler on the command line now:
+
+    .. code-block:: powershell
+
+      nvcc --version
+
+Invoking the compiler manually
+------------------------------
+
+To compile and link a single-file application, use the following commands:
+
+.. tab-set::
+  .. tab-item:: Linux and AMD
+    :sync: linux-amd
+
+    .. code-block:: bash
+
+      amdclang++ ./HIP-Basic/saxpy/main.hip -o saxpy -I ./Common -lamdhip64 -L /opt/rocm/lib -O2
+
+  .. tab-item:: Linux and NVIDIA
+    :sync: linux-nvidia
+
+    .. code-block:: bash
+
+      nvcc ./HIP-Basic/saxpy/main.hip -o saxpy -I ./Common -I /opt/rocm/include -O2 -x cu
+
+  .. tab-item:: Windows and AMD
+    :sync: windows-amd
+
+    .. code-block:: powershell
+
+      clang++ .\HIP-Basic\saxpy\main.hip -o saxpy.exe -I .\Common -lamdhip64 -L ${env:HIP_PATH}lib -O2
+
+  .. tab-item:: Windows and NVIDIA
```

Comment:
Have you tested this particular config? windows + nvidia?

Reply:
As I know, all the commands were tested. I can test this, today. Do you so problems with it?

Reply:
I never really tested this config so was not sure. If it works I am more than OK with it.